test/spec/app_spec.rb

```diff
@@ -94,28 +53,115 @@ def login
     window = @driver.window_handles.first
     @driver.switch_to.window(window)
     @driver.navigate.refresh
-    sleep 3
+    wait_for_selector('#save-button')
     expect(@driver.page_source.include?('sign in')).to be(false)
   end
 
-  def save_screenshot(title)
-    if @config['imgur_client_id']
-      path = '/tmp/' + (0...8).map{ (65 + rand(26)).chr }.join + '.png'
-      @driver.save_screenshot(path)
+  it 'should create media' do
+    login(media_type:"url",media_content:"https://meedan.com")
```

Comment:
Until we have a linter in place, I'll make those comments manually hehe... please add a space after the `,` and after the `:`'s, so it will look like this: `media_type: 'url', media_content: 'https://meedan.com'`